### PR TITLE
task 8 complete

### PR DESCRIPTION
**Pull request by parisselbygrace**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -249,3 +249,73 @@ return request(app)
...
+        .then(({ body }) => {
+          expect(body.article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 99,
+          });
```
Comment (xViolaine) [inline __tests__/app.test.js:286]: Nice job also testing if the negative works

File: app.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const express = require("express");
 const app = express();
 const {getTopics} = require ("./contorllors/getTopics");
-const {getArticles, getArticleById, getArticleCommentsById} = require ("./contorllors/articles");
-const { postComment } = require("./contorllors/articles");
+const {getArticles, getArticleById, getArticleCommentsById, postComment, patchVotes} = require ("./contorllors/articles");
```
Comment (xViolaine) [inline app.js:4]: just as a headsup that there was a spelling error when creating the controllers folder - as long as it's consistent like here when requiring in, it's technically fine though

File: __tests__/app.test.js [hunk 3]
```diff
@@ -249,3 +249,73 @@ return request(app)
...
+    test("400: if passed wrong data type", () => {
+      const votes = { inc_votes: "Hi" }
+      return request(app)
+        .patch("/api/articles/1")
+        .send(votes)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid data")
+        })
+    })
```
Comment (xViolaine) [inline __tests__/app.test.js:318]: What if you passed votes with a missing body?

**Review by xViolaine — CHANGES_REQUESTED** [review]
Another test required to check what happens when we try to pass votes with a missing body - just to make sure its all properly error handled
